lib/hawkular/hawkular_client.rb

```diff
@@ -66,7 +66,7 @@ def tokens
 
     # this is in a dedicated method, because constructor opens the websocket connection to make the handshake
     def init_operations_client
-      Operations::OperationsClient.new(entrypoint: @state[:entrypoint],
+      Operations::Client.new(entrypoint: @state[:entrypoint],
```

Comment:
It just worries me that Inventory has a `#create` method but is being used like Operation's constructor. 
Do you think it would confuse users? Maybe not.. but though on asking.